src/toolbar/tests/functional/Toolbar.ts

```diff
@@ -0,0 +1,47 @@
+const { registerSuite } = intern.getInterface('object');
+const { assert } = intern.getPlugin('chai');
+import { Remote } from 'intern/lib/executors/Node';
+
+function getPage(remote: Remote) {
+	return remote
+		.get('http://localhost:9000/_build/common/example/?module=toolbar')
+		.setFindTimeout(5000);
+}
+
+const DELAY = 400;
+const HEIGHT = 500;
+const WIDTH = 500;
+
+registerSuite('Toolbar', {
+	'Should show menu when button is clicked'() {
+		return getPage(this.remote)
+			.setWindowSize(WIDTH, HEIGHT)
+			.findByCssSelector('body > div:last-child > div:first-child > button')
+				.click()
+			.end()
+			.sleep(DELAY)
+			.findByCssSelector('body > div:last-child > div:first-child > div:nth-child(2) > div')
+				.isDisplayed()
+				.then(displayed => {
+					assert.isTrue(displayed);
+				});
+	},
+
+	'Should close menu when button is clicked'() {
+		return getPage(this.remote)
+			.setWindowSize(WIDTH, HEIGHT)
+			.findByCssSelector('body > div:last-child > div:first-child > button')
+				.click()
+			.end()
+			.sleep(DELAY)
+			.findByCssSelector('body > div:last-child > div:first-child > div:nth-child(2) > div button')
+				.click()
+			.end()
+			.sleep(DELAY)
+			.findByCssSelector('body > div:last-child > div:first-child > div:nth-child(2) > div')
+				.isDisplayed()
+				.then(displayed => {
+					assert.isFalse(displayed);
+				});
+	}
```

Comment:
I know functional tests are a pain, but maybe one to make sure the slidepane is shown or not shown based on window width would be good

Reply:
^ ignore that if you want, or if you already tried and removed it in frustration

Reply:
I think this is a good idea, but I had the hardest time getting functional tests to run locally. I can open a new issue for this if that works for you.